src/components/CookieWarning.js

```diff
@@ -55,7 +54,9 @@ const CookieWarning = () => {
       const cookieName = eqPos > -1 ? cookie.substr(0, eqPos) : cookie
       document.cookie = `${cookieName}=;expires=Thu, 01 Jan 1970 00:00:00 GMT`
     }, cookies)
-    signOut()
```

Comment:
We no longer want to sign out, because we want users to be able re-accept cookies, and if they are signed out we don't know which user it is who has accepted cookies.

Reply:
But if we don't sign them out, doesn't that mean that they can continue using Terra even without accepting the use of cookies?

Reply:
If we use the modal and block clicks with the overlay, they either need to jump through some large hoops to keep using Terra (go into the console and delete the modal, which I'd argue is equivalent to hitting ok), or agree to cookies. So it blocks use of the site through a different mechanism